catalog/spatial/registry/registry-common/src/main/java/org/codice/ddf/registry/common/RegistryConstants.java

```diff
@@ -24,23 +26,29 @@
 
     public static final String XML_BOUNDS_TYPE = "GM_Envelope";
 
+    public static final String REGISTRY_ID_PROPERTY = "org.codice.ddf.registry.registry-id";
```

Comment:
This does not follow the new taxonomy.  Seems like it should be something like `registry.id`.  Thoughts @jlcsmith and @brendan-hofmann?

https://codice.atlassian.net/wiki/display/DDF/Attribute+Taxonomy+and+Mappings